services/web-app/components/card-group/card-group.module.scss

```
@@ -0,0 +1,24 @@
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+// Sets gray border on the top of cards at the smallest breakpoint and removes the margin set by the <Column> in the Grid component.
+.card-group [class*='column'] {
+  border-top: 1px solid theme.$border-subtle-01;
+  margin-bottom: 0;
+}
+
+.card-group [class*='column']:first-child {
+  border-top: 0 solid transparent;
+}
+
+// Borders for cards at the medium breakpoint
+.card-group [class*='column']:nth-child(-n + 2) {
+  @include breakpoint.breakpoint('md') {
+    border-top: 0 solid transparent;
+  }
+}
+
+.card-group [class*='column']:nth-child(even) {
+  @include breakpoint.breakpoint('md') {
+    border-left: 1px solid theme.$border-subtle-01;
```

```suggestion
    box-shadow: -1px 0 0 0 theme.$border-subtle;
```
this should fix the extra pixel issue @andreancardona was showing

---

thanks!!